From 332bc3b328df31985e216bb4305284841ed3a3ac Mon Sep 17 00:00:00 2001
From: endaaman <buhibuhidog@gmail.com>
Date: Thu, 12 Dec 2019 19:43:53 +0900
Subject: [PATCH 2/3] futures.CancelledError -> asyncio.CancelledError

---
 gbulb/transports.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/gbulb/transports.py b/gbulb/transports.py
index ebc9f81..c91c6e3 100644
--- a/gbulb/transports.py
+++ b/gbulb/transports.py
@@ -1,6 +1,7 @@
 import collections
 import socket
 import subprocess
+import asyncio
 from asyncio import base_subprocess, futures, transports
 
 
@@ -159,7 +160,7 @@ class ReadTransport(BaseTransport, transports.ReadTransport):
             self._force_close(exc)
         except OSError as exc:
             self._fatal_error(exc, 'Fatal read error on pipe transport')
-        except futures.CancelledError:
+        except asyncio.CancelledError:
             if not self._closing:
                 raise
         except futures.InvalidStateError:
-- 
2.25.4

